SparkPost/2web2ui
# EO-812: Added injections column to health score overview

### What Changed
 - Added injections column to health score page

### How To Test
 - [Point openresty to prd](https://confluence.int.messagesystems.com/pages/viewpage.action?spaceKey=ENG&title=Configuring+Openresty+Upstreams)
- Check the health score page and see the new column after `Current Score`
 - Select a custom date before 7 Jun 2019 (new model date)
 - Column should not be sortable and it should render `- - -` instead

### To Do
